### ISSUE TITLE
[Feature Request] Timer in Flow Condition

### ISSUE THREAD

**Issue by OliverIckler** [athombv/homey#2366]
It would be a great afford if it is possible to configure a timer within a flow condition.
The THEN CLAUSE should be called when the condition is true for the defined time.

In devolo home control you can specify the timing on every card.
![devolo](https://user-images.githubusercontent.com/37276327/37852851-7d4a9924-2ee4-11e8-9403-5f8edad271a9.png)

**Comment by zenturacp** [comment]
Quite easy to achieve with Countdown?

**Comment by OliverIckler** [comment]
Yes, countdown will work, but i have more than 40 plugs currently in my devolo manager.
Adding 2 countdowns for every device will be very confusing to maintenance.
Also there are 2 countdowns for every window sensor to decrease and raise temperature, when opening, after 5 minutes.

I think it would be very useful to see how long the current state of a capability is set.

So Countdown is a solution that may work if you want to start with smart home.
If you want to automate your complete home, it is not comfortable!

Currently i am still using devolo manager for my door sensors and plugs instead of homey, although homey has great features. But it seems some basics are really missing.

**Comment by OliverIckler** [comment]
I testet it with homey.

On my devolo I have 1 flow:
If window is open for 5 minutes then decrase temp, after that set back to original value

On homey i added:
First flow: If windows is open then start countdown timer
Second flow: If windows is closed then stop countdown timer
Third flow: store temp to variable, decrease temp, activate fourth flow, deactivate second flow
First Countdown: Start third flow after 5 minutes
Fourth flow: if window is closed then restore temp from variable, deactivate fourth flow, activate first flow

Does anyone has a better idea?
